aya/src/bpf.rs

```diff
@@ -109,6 +114,133 @@ pub fn features() -> &'static Features {
     &FEATURES
 }
 
+lazy_static! {
+    static ref KCONFIG_DEFINITION: HashMap<String, Vec<u8>> = compute_kconfig_definition(&FEATURES);
```

Comment:
Instead of this lazy static, we could have a KConfig type. Then we could have
`EbpfLoader::with_kconfig(kconfig)`. This way we don't force people who don't
use kconfig to take the extra memory hit. It's a small hit, but people are using
aya in embedded systems so we shouldn't increase mem usage if we can.